RecoBTag/SecondaryVertex/python/secondaryVertex_cff.py

```diff
@@ -46,8 +46,10 @@
 from RecoBTag.SecondaryVertex.pfGhostTrackVertexTagInfos_cfi import *
 from RecoBTag.SecondaryVertex.pfGhostTrackBJetTags_cfi import *
 from RecoBTag.SecondaryVertex.pfInclusiveSecondaryVertexFinderAK8TagInfos_cfi import *
+from RecoBTag.SecondaryVertex.pfBoostedDoubleSVAK8TagInfos_cfi import *
 from RecoBTag.SecondaryVertex.pfBoostedDoubleSecondaryVertexAK8BJetTags_cfi import *
 from RecoBTag.SecondaryVertex.pfInclusiveSecondaryVertexFinderCA15TagInfos_cfi import *
+from RecoBTag.SecondaryVertex.pfBoostedDoubleSVCA15TagInfos_cfi import *
```

Comment:
I guess you are right since these are mostly cfi files defining single modules. On the other hand, the b tagging code is riddled with `import *` statements and fixing all of them is beyond the scope of this PR. So for now left as is.